console/console-init/ui/src/modules/connection-detail/components/ConnectionLinksList/ConnectionLinksList.tsx

```diff
@@ -36,7 +36,7 @@ export interface ILink {
   status?: "creating" | "deleting" | "running";
 }
 
-export const LinkList: React.FunctionComponent<ILinkListProps> = ({
+export const ConnectionLinksList: React.FunctionComponent<IConnectionLinksListProps> = ({
```

Comment:
at some places ConnectionLinks is replaced by ConnectionDetail. I think ConnectionLinks sounds more accurate

Reply:
I did that intentionally as I wanted to differentiate between the specifics of the page which represent links and the part which represent connection specific details.